Sources/App/Setup/boot.swift

```diff
@@ -0,0 +1,12 @@
+import Sugar
+import Vapor
+
+/// Called after your application has initialized.
+public func boot(_ app: Application) throws {
+    // MARK: Leaf tags
+    // Add your tags like this:
+    // let tags = try app.make(MutableLeafTagConfig.self)
```

Comment:
Just to be sure, we're still recommending the MutableLeafTagConfig? Not the `use...Tags` pattern?

Reply:
Just pushed the update to use the `use...Tags` pattern (after all packages got updated) 🎉